Library/Formula/hue.rb

```diff
@@ -0,0 +1,157 @@
+-        'tablib.packages',
++packages = [
++    'tablib', 'tablib.formats',
++    'tablib.packages',
++    'tablib.packages.omnijson',
++    'tablib.packages.unicodecsv'
++]
++
++if sys.version_info[0] == 2:
++    packages.extend([
+         'tablib.packages.xlwt',
+-        'tablib.packages.xlwt3',
+         'tablib.packages.xlrd',
+-        'tablib.packages.xlrd3',
+-        'tablib.packages.omnijson',
+         'tablib.packages.odf',
+-        'tablib.packages.odf3',
+         'tablib.packages.openpyxl',
+         'tablib.packages.openpyxl.shared',
+         'tablib.packages.openpyxl.reader',
+         'tablib.packages.openpyxl.writer',
++        'tablib.packages.yaml'
++    ])
++else:
++    packages.extend([
++        'tablib.packages.xlwt3',
++        'tablib.packages.xlrd3',
++        'tablib.packages.odf3',
+         'tablib.packages.openpyxl3',
+         'tablib.packages.openpyxl3.shared',
+         'tablib.packages.openpyxl3.reader',
+         'tablib.packages.openpyxl3.writer',
+-        'tablib.packages.yaml',
+-        'tablib.packages.unicodecsv'
+-    ],
++        'tablib.packages.yaml3'
++    ])
++
++setup(
++    name='tablib',
++    version=tablib.__version__,
++    description='Format agnostic tabular data library (XLS, JSON, YAML, CSV)',
++    long_description=(open('README.rst').read() + '\n\n' +
++        open('HISTORY.rst').read()),
++    author='Kenneth Reitz',
++    author_email='me@kennethreitz.com',
++    url='http://python-tablib.org',
++    packages=packages,
+     license='MIT',
+     classifiers=(
+         'Development Status :: 5 - Production/Stable',
+
+diff --git a/desktop/core/ext-py/parquet-python/parquet/bitstring.py b/desktop/core/ext-py/parquet-python/parquet/bitstring.py
+index ab807a2..7203c94 100644
+--- a/desktop/core/ext-py/parquet-python/parquet/bitstring.py
++++ b/desktop/core/ext-py/parquet-python/parquet/bitstring.py
+@@ -1,7 +1,5 @@
+-
+ SINGLE_BIT_MASK =  [1 << x for x in range(7, -1, -1)]
+ 
+-]
+ 
+ class BitString(object):
+ 
+@@ -10,7 +8,6 @@ class BitString(object):
+ 		self.offset = offset if offset is not None else 0
+ 		self.length = length if length is not None else 8 * len(data) - self.offset 
+ 
+-
+ 	def __getitem__(self, key):
+ 		try:
+ 			start = key.start
+@@ -18,5 +15,5 @@ class BitString(object):
+ 		except AttributeError:
+ 			if key < 0 or key >= length:
+ 				raise IndexError()
+-			byte_index, bit_offset = divmod(self.offset + key), 8)
+-			return self.bytes[byte_index] & SINGLE_BIT_MASK[bit_offset]
+\ No newline at end of file
++			byte_index, bit_offset = (divmod(self.offset + key), 8)
++			return self.bytes[byte_index] & SINGLE_BIT_MASK[bit_offset]
+diff --git a/Makefile.vars.priv b/Makefile.vars.priv
+index 79bc443..e17e192 100644
+--- a/Makefile.vars.priv
++++ b/Makefile.vars.priv
+@@ -53,7 +53,7 @@ BLD_DIR_DOC := $(BLD_DIR)/docs
+ 
+ # Installation directory
+ PREFIX ?= /usr/local
+-INSTALL_DIR ?= $(PREFIX)/hue
++INSTALL_DIR ?= $(PREFIX)/libexec
```

Comment:
Yeah, that was the first thing I tried.  When I set INSTALL_DIR from the command line, I get the following error when the first Hue web submodule is installed:

```
--- Making built distribution at /private/tmp/hue20150531-47630-1au188t/hue-release-3.8.1/desktop/core/build/bdist/desktop-3.8.1
--- Install built distribution for desktop at /usr/local/Cellar/hue/3.8.1/libexec
ERROR: /usr/local/Cellar/hue/3.8.1/libexec not empty. Cowardly refusing to continue.
make[2]: *** [install-bdist] Error 1
make[1]: *** [.recursive-install-bdist/core] Error 2
make: *** [install-desktop] Error 2
```

It looks like this is happening inside this target:

https://github.com/cloudera/hue/blob/master/Makefile.sdk#L245

So, I'd need to patch anyways.

I'm going to open an issue with Hue.  For now, patching the INSTALL_DIR variable instead of setting it on the command line works fine.